meetings/bordeaux2018/index.md

```diff
@@ -48,12 +48,12 @@ A preliminary schedule is as follows.
 
 ## Committee
 
-- Miles Lubin
-- Oscar Dowson
-- Benoît Legat
-- Joaquim Garcia
-- Juan Pablo Vielma
-- Issam Tahiri (Local Organization)
-- François Vanderbeck (Local Organization)
+- Miles Lubin (Google)
+- Oscar Dowson (U. of Auckland)
+- Benoît Legat (UCL)
```

Comment:
Could you change UCL to UCLouvain ? (UCL can be confused with University College de Londres)